### ISSUE TITLE
Trailing whitespace in COPY causes uncaught error

### ISSUE THREAD

**Issue by lindhe** [hadolint/hadolint#448]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Either a warning for trailing whitespace, or that it ignores them altogether.

### Actual behavior
```shell
hadolint: Data.Text.last: empty input
CallStack (from HasCallStack):
  error, called at libraries/text/Data/Text.hs:1858:18 in text-1.2.4.0:Data.Text
```

### Steps to reproduce the behavior

Create a Dockerfile with a COPY directive that has a trailing whitespace, and run `hadolint` on it.

**Output of `hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.18.0-0-g76eee5c
```

**Dockerfile**

Note that there is a trailing whitespace on line 2!
```Dockerfile
FROM scratch
COPY a b 
```

**Additional environment details (OS, stack version, etc.)**
Ubuntu 20.04
